src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowJob.java

```diff
@@ -138,6 +140,9 @@ public WorkflowJob(ItemGroup parent, String name) {
         for (Trigger t : triggers) {
             t.start(this, Items.currentlyUpdatingByXml());
         }
+        if (concurrentBuild != null) {
```

Comment:
Not happy about this, but `setConcurrentBuild(boolean)` alone wasn't enough to get the migration working.